File: data/code_pairs/fortran/940.f90
module dft_setting_params
    implicit none

    integer, parameter :: max_iter = 100
    real(kind=8), parameter :: conv_thresh = 1.0e-6
    logical :: print_info = .true.
#ifdef __LIBXC
    character(len=10) :: xc_functional = 'PBE'
#else
    character(len=10) :: xc_functional = 'LDA'
#endif

end module dft_setting_params

File: data/code_pairs/cpp/940.cpp
#ifndef DFT_SETTING_PARAMS_HPP
#define DFT_SETTING_PARAMS_HPP

#include <string>

namespace dft_setting_params {

    constexpr int max_iter = 100;
    constexpr double conv_thresh = 1.0e-6;
    bool print_info = true;

#ifdef __LIBXC
    std::string xc_functional = "PBE";
#else
    std::string xc_functional = "LDA";
#endif

} // namespace dft_setting_params

#endif // DFT_SETTING_PARAMS_HPP